platform-enterprise_versioned_docs/version-25.2/studios/overview.md

```diff
@@ -199,6 +191,61 @@ The maximum storage allocation for a session is limited by the compute environme
 
 Stop the active session to trigger a snapshot from the active volume. The snapshot is uploaded to cloud storage with Fusion. When you start from the newly saved snapshot, all previous data is loaded, and the newly started session will have 2 GB of available space.
 
+## Limitations
+
+### EFS file systems
+
+If you configured your compute environment to include an EFS file system with **EFS file system > EFS mount path**, the mount path must be explicitly specified. The mount path cannot be the same as your compute environment work directory. If the EFS file system is mounted as your compute environment work directory, snapshots cannot be saved and sessions fail. 
+
+To mount an EFS volume in a Studio session (for example, if your organization has a custom, managed, and standardized software stack in an EFS volume), add the EFS volume to the compute environment (system ID and mount path). The volume will be available at the specified mount path in the session.
+
+For more information on AWS Batch configuration, see [AWS Batch][aws-batch].
+
+### Path-based routing/non-wildcard SSL/TLS certificates
+
+Connect, the Studios webserver, uses dynamic subdomains to manage session routing of requests (for example https://a1234abc.connect.cloud.seqera.io/, https://a5678abcd.connect.cloud.seqera.io/), which requires a wildcard SSL/TLS certificates. 
+
+For Enterprise deployments that cannot use a wildcard SSL/TLS certificate, an optional configuration environment variable, `TOWER_DATA_STUDIO_ENABLE_PATH_ROUTING` can be added. Setting this configures Studios requests to use path-based routing and a single, fixed domain for Studio sessions (for example https://connect.connect.cloud.seqera.io/_studio/a1234abc, https://connect.connect.cloud.seqera.io/_studio/a5678abcd).
```

Comment:
Why the `connect.connect`? Surely it's just singular and prefixed with the `{sessionId}`?

```suggestion
For Enterprise deployments that cannot use a wildcard SSL/TLS certificate, an optional configuration environment variable, `TOWER_DATA_STUDIO_ENABLE_PATH_ROUTING` can be added. Setting this configures Studios requests to use path-based routing and a single, fixed domain for Studio sessions (for example https://connect.cloud.seqera.io/_studio/a1234abc, https://connect.cloud.seqera.io/_studio/a5678abcd).
```